AutoCollection/Performance.ts

```diff
@@ -17,6 +17,8 @@ class AutoCollectPerformance {
     private _handle: NodeJS.Timer;
     private _isEnabled: boolean;
     private _isInitialized: boolean;
+    private _lastAppCpuUsage: { user: number, system: number } = {user: 0, system: 0};
+    private _lastHrtime: number[] = process.hrtime();
```

Comment:
Probably don't want to record this until `enable` is called. Otherwise the first sample will be over a very long time range

Reply:
Moved to where other _lastX vars are set to initial condition